mappings/net/minecraft/util/collection/ImmutablePool.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_6005 net/minecraft/util/collection/ImmutablePool
+	METHOD method_34971 builder ()Lnet/minecraft/class_6005$class_6006;
+	METHOD method_34972 decode (Lcom/mojang/serialization/Codec;)Lcom/mojang/serialization/Codec;
```

Comment:
This is definitely wrong. It is creating a codec of the pool from the pool itself